doc/api/cli.md

```diff
@@ -194,6 +194,19 @@ added: v0.10
 Aborting instead of exiting causes a core file to be generated for post-mortem
 analysis using a debugger (such as `lldb`, `gdb`, and `mdb`).
 
+### `--abort-on-unhandled-rejection`
+<!-- YAML
+added: REPLACEME
+-->
+
+Aborting instead of exiting causes a core file to be generated for post-mortem
+analysis using a debugger (such as `lldb`, `gdb`, and `mdb`).
+
+This option only affects `Promise` rejections. It is only available on POSIX
+systems. It is implemented by keeping `fork()`ed copies of the process alive
+until it is known whether the `Promise` is handled within the same eventloop
+iteration or is left unhandled, which may come with performance implications.
```

Comment:
With the current wording it's somewhat ambiguous if 'performance implications' applies to forking or unhandled rejections.